### DISCUSSION TITLE
Add a way to join channel memberships for free

### DISCUSSION THREAD

**Discussion by Opulent54** [ReVanced/revanced-discussions#293] [Question] [answered]
(if possible) Add a way to join channel memberships for free below 5$

**Comment by Sojaquad** [comment, marked as answer]
I'd say this is a good idea, but I'm almost certain it's impossible. It's entirely possible to modify the user's interface and how they watch videos, but it's almost impossible to change how other people see them, and what priveleges they get on the site. With a channel membership, you need to pay in order to get the features, as it is done on YouTube's servers, which we do not have access to. You theoretically _could_ make them show up as a member to other ReVanced people, but that's likely not happening.

**Reply by gulu43** [reply]
Thank you for Confirmation 👍🏻

**Comment by TheJeterLP** [comment]
Not possible to do. This is handled by server side and cannot be manipulated by us.
